orttraining/orttraining/training_ops/cuda/math/batch_scale.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#include "core/providers/cuda/cuda_kernel.h"
+
+namespace onnxruntime {
+namespace cuda {
+
+class BatchScale final : public CudaKernel {
+ public:
+  BatchScale(const OpKernelInfo& info) : CudaKernel(info) {
+    ORT_ENFORCE(info.GetAttr<float>("scale_0", &scale_0_).IsOK());
+    ORT_ENFORCE(info.GetAttr<float>("scale_1", &scale_1_).IsOK());
+    info.GetAttrOrDefault<float>("scale_2", &scale_2_, -1.f);
```

Comment:
Does this op have the constraint the scale > 0 ? Need to add a comment to its definition?

Reply:
This is incorrect, I have remove using -1 as default values.